src/redux/throttleMiddleware.test.ts

```diff
@@ -53,7 +53,6 @@ describe("throttleMidddlware", (): void => {
   });
   it("dispatches ValuesChanged when receiving ValueChanged", (): void => {
```

Comment:
I'm not really sure what this test is trying to show now.

Reply:
Have changed it now so the test manually triggers the queue flushing.